app/src/main/java/com/arctouch/codechallenge/data/model/Genre.java

```diff
@@ -1,6 +1,8 @@
-package com.arctouch.codechallenge.model;
+package com.arctouch.codechallenge.data.model;
 
-public class Genre {
+import java.io.Serializable;
+
+public class Genre implements Serializable {
```

Comment:
Why did you use `Serializable` instead of `Parcelable`?

Reply:
Because I only needed to pass this object for the other activity through the bundle. And the object is not big so the impact in using Serializable is low. But I agree that using Parcelable is better because is more optimized for Android.